### PR TITLE
fix bundle for upstream

### PR DESCRIPTION
**Pull request by jcantrill**
### Description
This PR fixes bundle generation for upstream 5.1

@vimalk78 I thought you already had a PR to do this?  did we have a bad merge?

### CODE REVIEW

File: bundle.Dockerfile [hunk 1]
```diff
@@ -4,14 +4,14 @@ LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
...
-LABEL operators.operatorframework.io.bundle.channel.default.v1=5.0
+LABEL operators.operatorframework.io.bundle.channels.v1=5.1
+LABEL operators.operatorframework.io.bundle.channel.default.v1=5.1
 
 COPY bundle/manifests /manifests/
 COPY bundle/metadata /metadata/
 
 LABEL com.redhat.delivery.operator.bundle=true
-LABEL com.redhat.openshift.versions="v5.0"
+LABEL com.redhat.openshift.versions="v5.1"
```
Comment (vimalk78) [inline bundle.Dockerfile:14]: this should be `4.7`

File: Makefile [hunk 2]
```diff
@@ -168,7 +168,7 @@ test-cluster:
 
 MANIFEST_VERSION?="5.0"
```
Comment (vparfonov) [inline Makefile:169]: 5.1 ?

Comment (jcantrill) [reply Makefile:169]: removing...

**Pull request merged**